frontend/app/assets/svg.tsx

```diff
@@ -246,3 +246,47 @@ export const SVGCloseIcon = ({ className }: { className: string }) => (
     <path d="M12.0008 12.7269L8.40884 16.3189C8.30217 16.4256 8.18417 16.4756 8.05484 16.4689C7.9255 16.4622 7.8075 16.4059 7.70084 16.2999C7.59417 16.1939 7.54084 16.0726 7.54084 15.9359C7.54084 15.8006 7.59417 15.6796 7.70084 15.5729L11.2738 11.9999L7.68184 8.4329C7.57517 8.32623 7.52517 8.2049 7.53184 8.0689C7.5385 7.9329 7.59484 7.8119 7.70084 7.7059C7.80684 7.5999 7.92817 7.54657 8.06484 7.5459C8.20017 7.5459 8.32117 7.59923 8.42784 7.7059L12.0008 11.2979L15.5678 7.7059C15.6745 7.59923 15.7925 7.5459 15.9218 7.5459C16.0512 7.5459 16.1692 7.59923 16.2758 7.7059C16.3865 7.8159 16.4418 7.9379 16.4418 8.0719C16.4418 8.2059 16.3865 8.3259 16.2758 8.4319L12.7028 11.9999L16.2948 15.5919C16.4015 15.6986 16.4548 15.8166 16.4548 15.9459C16.4548 16.0752 16.4015 16.1932 16.2948 16.2999C16.1848 16.4106 16.0628 16.4659 15.9288 16.4659C15.7948 16.4659 15.6748 16.4106 15.5688 16.2999L12.0008 12.7269Z"></path>
   </svg>
 )
+
+export const SVGLinkIcon = ({ className }: { className: string }) => (
+  <svg className={className} viewBox="0 0 20 20" fill="none">
+    <path
+      d="M8.84667 13.3974H5.8975C4.9575 13.3974 4.15639 13.0663 3.49417 12.4041C2.83195 11.7418 2.50056 10.9405 2.5 9.9999C2.49945 9.05934 2.83083 8.25795 3.49417 7.59573C4.1575 6.93351 4.95861 6.60212 5.8975 6.60156H8.84667V7.4349H5.8975C5.19195 7.4349 4.58833 7.68601 4.08667 8.18823C3.58444 8.69101 3.33333 9.2949 3.33333 9.9999C3.33333 10.7049 3.58444 11.3085 4.08667 11.8107C4.58889 12.313 5.1925 12.5641 5.8975 12.5641H8.84667V13.3974ZM7.08333 10.4166V9.58323H12.9167V10.4166H7.08333ZM11.1542 13.3974V12.5641H14.1025C14.8081 12.5641 15.4117 12.313 15.9133 11.8107C16.4156 11.3085 16.6667 10.7049 16.6667 9.9999C16.6667 9.2949 16.4156 8.69128 15.9133 8.18906C15.4111 7.68684 14.8075 7.43573 14.1025 7.43573H11.1542V6.6024H14.1025C15.0425 6.6024 15.8439 6.93351 16.5067 7.59573C17.1694 8.25795 17.5006 9.05934 17.5 9.9999C17.4994 10.9405 17.1681 11.7418 16.5058 12.4041C15.8436 13.0663 15.0425 13.3977 14.1025 13.3982L11.1542 13.3974Z"
+      fill="white"
+    />
```

Comment:
This is not optimized SVG in case:
```suggestion
    <path fill="currentColor" d="M8.847 13.397h-2.95c-.94 0-1.74-.33-2.403-.993A3.278 3.278 0 0 1 2.5 10c0-.94.33-1.742.994-2.404a3.28 3.28 0 0 1 2.403-.994h2.95v.833h-2.95c-.705 0-1.309.251-1.81.753A2.472 2.472 0 0 0 3.333 10c0 .705.251 1.309.754 1.81a2.469 2.469 0 0 0 1.81.754h2.95v.833Zm-1.764-2.98v-.834h5.834v.834H7.083Zm4.071 2.98v-.833h2.948c.706 0 1.31-.251 1.811-.753A2.469 2.469 0 0 0 16.667 10c0-.705-.251-1.309-.754-1.81a2.469 2.469 0 0 0-1.81-.754h-2.949v-.834h2.948c.94 0 1.742.332 2.405.994A3.27 3.27 0 0 1 17.5 10c0 .94-.332 1.742-.994 2.404a3.276 3.276 0 0 1-2.404.994h-2.948Z"/>
```